src/clients/meta/MetaClient.cpp

```diff
@@ -738,11 +738,13 @@ void MetaClient::getResponse(Request req,
               }
 
               auto&& resp = t.value();
-              if (resp.get_code() == nebula::cpp2::ErrorCode::SUCCEEDED) {
+              auto code = resp.get_code();
+              if (code == nebula::cpp2::ErrorCode::SUCCEEDED) {
                 // succeeded
                 pro.setValue(respGen(std::move(resp)));
                 return;
-              } else if (resp.get_code() == nebula::cpp2::ErrorCode::E_LEADER_CHANGED) {
+              } else if (code == nebula::cpp2::ErrorCode::E_LEADER_CHANGED ||
+                         code == nebula::cpp2::ErrorCode::E_MACHINE_NOT_FOUND) {
```

Comment:
Em... if you modify in this way, how to distinguish from actual `MACHINE_NOT_FOUND` and `MACHINE_NOT_FOUND` with LEADER_CHANGE